csi.proto

```diff
@@ -227,6 +206,15 @@ message PluginCapability {
       // returned by NodeGetInfo to ensure that a given volume is
       // accessible from a given node when scheduling workloads.
       VOLUME_ACCESSIBILITY_CONSTRAINTS = 2;
+
+      // GROUP_CONTROLLER_SERVICE indicates that the Plugin provides
+      // RPCs for the GroupControllerService. Plugins MAY provide this
+      // capability.
+      // The presence of this capability determines whether the CO will
+      // attempt to invoke the REQUIRED GroupControllerService RPCs, as
+      // well as specific RPCs as indicated by
+      // GroupControllerGetCapabilities.
+      GROUP_CONTROLLER_SERVICE = 3;
```

Comment:
@xing-yang this is better abstraction ! just wondering do we need to explicitly state this `group` is for snapshot or volumes ? because we may have different groups like volumes, snapshots and something tomorrow like just for example : `dataSource`?

Reply:
We'll have too many services if one service is for one group type.